sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -460,13 +486,21 @@ private void setReadinessCheck(
             Protos.TaskInfo.Builder taskInfoBuilder,
             String serviceName,
             PodInstance podInstance,
-            TaskSpec taskSpec) {
+            TaskSpec taskSpec,
+            GoalStateOverride override) {
         if (!taskSpec.getReadinessCheck().isPresent()) {
             LOGGER.debug("No readiness check defined for taskSpec: {}", taskSpec.getName());
             return;
         }
 
+
         ReadinessCheckSpec readinessCheckSpec = taskSpec.getReadinessCheck().get();
+        String readinessCheckCmd = readinessCheckSpec.getCommand();
+
+        if (override.equals(GoalStateOverride.STOPPED)) {
```

Comment:
Reordering the conditional as:
```java
 if (GoalStateOverride.STOPPED.equals(override)) {

}
```
will also handle `null` values for `override` correctly.

Reply:
`override` should never be null.  I did this on purpose.